addons/website_livechat/static/src/components/discuss/discuss.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-    <t t-inherit="mail.Discuss.content" t-inherit-mode="extension">
-        <xpath expr="//*[hasclass('o_Discuss_thread')]" position="before">
-            <t t-if="discuss.thread.visitor">
+    <t t-inherit="mail.ThreadView" t-inherit-mode="extension">
+        <xpath expr="//*[hasclass('o_ThreadView_threadViewTopbar')]" position="after">
+            <t t-if="threadView.thread.visitor">
```

Comment:
With this change, `VisitorBanner` is shown in chat window.
This is a problem because space is precious in chat window, and visitor banner will make message list hard to read.
So instead of just condition on `thead.visitor`, make a field in model `ThreadView` (or `mail.thread` in JS with 15.0 syntax) that conditionally show VisitorBanner in Discuss app (i.e. the `ThreadView` record should be in the context of `Discuss`, i.e. there is a relational field that is set between them. See `ThreadView/threadViewer` and `ThreadViewer/discuss` fields).